spec/unit/report/audit_report_spec.rb

```diff
@@ -58,6 +58,13 @@
   end
 
   describe 'get_opts method' do
+    let(:mynode) { Chef::Node.new }
+
+    before :each do
+      mynode.default['audit']['inspec_backend_cache'] = true
```

Comment:
Should we add a test that assures that `opts[:backend_cache]` gets set to false if the node attribute calls for it? I hate to only have a test for the default path.